python/lsst/sims/catUtils/utils/CatalogTestUtils.py

```diff
@@ -370,7 +370,7 @@ def get_cartoon_agn_mags(self):
         indices = [ii for ii, name in enumerate(self.get_cartoon_agn_mags._colnames) \
                    if name in self._actually_calculated_columns]
 
-        if len(indices)==6:
+        if len(indices)==5:
```

Comment:
Looks like some repeated code and a random magic number ==5.  Maybe make a _getIndices method with the 5 as a default kwarg?